src/convertToObject.js

```diff
@@ -3,16 +3,29 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const sourceArr = sourceString.split(';');
+  const newObj = {};
+
+  for (const ch of sourceArr) {
+    const [key, value] = ch.split(':');
+
+    if (value !== undefined) {
```

Comment:
```suggestion
    if (value) {
```

Reply:
it gives the same result

Reply:
But what if there are only white spaces after splitting? Should we add these styles?

Reply:
> But what if there are only white spaces after splitting? Should we add these styles?

No, I think it's pointless to write down these styles